src/libraries/Microsoft.PowerFx.Core/Texl/BuiltinFunctionsCore.cs

```diff
@@ -76,6 +76,7 @@ internal class BuiltinFunctionsCore
         public static readonly TexlFunction Hour = _library.Append(new HourFunction());
         public static readonly TexlFunction If = _library.Append(new IfFunction());
         public static readonly TexlFunction IfError = _library.Append(new IfErrorFunction());
+        public static readonly TexlFunction IndexT = _library.Append(new IndexTFunction());
```

Comment:
If you're adding this to the library, make sure you create and get signoff on a pr in Canvas before checking this in. Otherwise, just add it at the bottom here, not as part of the library

Reply:
removed from library

Reply:
it looks like it's still in _library

Reply:
fixed